2020-02-Spring/1_strings/README.md

```diff
@@ -119,21 +138,270 @@ Go to [Top](#top)
 <!-- Don't remove -->
 <a name="s2"/>
 
-### 2. SOLUTION 2 TODO :bug:
+### 2. Sort Characters By Frequency Solution
 
-Source: TODO :bug:
+#### Algorithm Overview
 
-#### Naive/Simple Solution
+1. Count the number of times each character occurs in the string
+2. Group the characters into strings with identical characters
+3. Sort the strings by length in ascending order
+4. Create the result string
 
-TODO :bug:
+There are many ways that you can do this. I've covered two solutions that use maps and arrays respectively, but you could also solve this using a priority queue.
```

Comment:
```suggestion
There are many ways that you can do this. I've covered two solutions that use arrays and maps respectively, but you could also solve this using a priority queue.
```
It said maps and arrays respectively but it went like solution 1: arrays, solution 2: maps so it was a little bit confusing.

Reply:
Nice catch! :D Thanks